cem/utils.c

```diff
@@ -4,13 +4,13 @@
 #include "utils.h"
 
 // Allocate memory for a 2D matrix as a continuous block.
```

Comment:
This is one of two options, as far as I see things: 1) explicity specify pointer size during allocation or 2) use size of char* to do all the pointer arithmetic